internal/controller/hazelcast/hazelcast_resources.go

```diff
@@ -886,6 +886,36 @@ func (r *HazelcastReconciler) reconcileAgentConfig(ctx context.Context, h *hazel
 	})
 }
 
+func (r *HazelcastReconciler) reconcileLiteMemberCount(ctx context.Context, h *hazelcastv1alpha1.Hazelcast, logger logr.Logger) error {
+	if !(h.Spec.LiteMemberCount != nil && *h.Spec.LiteMemberCount > 0) {
+		return nil
+	}
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metadata(h),
+		Data:       make(map[string]string),
+	}
+	cm.Name = cm.Name + n.LiteSuffix
+	err := controllerutil.SetControllerReference(h, cm, r.Scheme)
+	if err != nil {
+		return fmt.Errorf("failed to set owner reference on ConfigMap: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to set owner reference on LiteConfigMap: %w", err)
```
Just `ConfigMap` could be confusing later.